tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -938,7 +938,7 @@ test.describe('Path /search/granules', () => {
         const body = request.postData()
 
         if (body) {
-          expect(body).toBe('end_date=2015-02-02T00:00:00.000Z&interval=hour&start_date=2014-12-04T00:00:00.000Z&concept_id[]=C194001210-LPDAAC_ECS')
+          expect(body).toBe('end_date=2015-02-07T00:00:00.000Z&interval=hour&start_date=2014-12-09T00:00:00.000Z&concept_id[]=C194001210-LPDAAC_ECS')
```

Comment:
Write new ticket for redux store changes to track specific collections' zoomLevel and center.  Skip this test, and comment.